src/cloud/project/project-conf-files_magento-app.md

```diff
@@ -18,12 +18,10 @@ Use the following properties to build your application configuration file. The `
 
 ### `name`
 
-{{site.data.var.ee}} supports multiple applications in a project, so you need a unique name that identifies the application in the project. You must use lower case alphanumeric characters, such as `a` to `z` and `0` to `9` for the name.
-
-The name is used in the [`routes.yaml`]({{ site.baseurl }}/cloud/project/project-conf-files_routes.html) file to define the HTTP upstream (by default, `php:http`). For example, if the value of `name` is `app`, you must use `app:http` in the upstream field. You can also use this name in multi-application relationships.
+The `name` property provides the application name used in the [`routes.yaml`]({{ site.baseurl }}/cloud/project/project-conf-files_routes.html) file to define the HTTP upstream (by default, `mymagento:http`). For example, if the value of `name` is `app`, you must use `app:http` in the upstream field. You can also use this name in multi-application relationships.
```

Comment:
> You can also use this name in multi-application relationships.

For me it looks like official approval of using multiple application on cloud environment.
I would remove it.